charts/thoras/templates/crd/aiscaletarget.yaml

```diff
@@ -89,6 +89,173 @@ spec:
                                 x-kubernetes-int-or-string: true
                         required:
                           - name
+                scale_metrics:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      type:
+                        type: string
+                        pattern: '^(Object|Resource|Pods|External)$'
+                      object:
+                        type: object
+                        required:
+                          - metric
+                          - describedObject
+                          - target
+                        properties:
+                          metric:
+                            type: object
+                            required:
+                              - name
+                            properties:
+                              name:
+                                type: string
+                              selector:
+                                type: string
+                          describedObject:
+                            type: object
+                            required:
+                              - kind
+                              - name
+                            properties:
+                              apiVersion:
+                                type: string
+                              kind:
+                                type: string
+                              name:
+                                type: string
+                          target:
+                            type: object
+                            required:
+                              - type
+                            properties:
+                              type:
+                                type: string
+                                pattern: '^(Utilization|Value|AverageValue)$'
+                              averageUtilization:
+                                anyOf:
+                                  - type: integer
+                                  - type: string
+                                x-kubernetes-int-or-string: true
+                              averageValue:
+                                anyOf:
+                                  - type: integer
+                                  - type: string
+                                x-kubernetes-int-or-string: true
+                              value:
+                                anyOf:
```

Comment:
[dust] According to chatgpt we only need `x-kubernetes-int-or-string` and not both that + anyOf